octoprint_enclosure/templates/enclosure_settings.jinja2

```diff
@@ -808,7 +808,7 @@
           <option value="high">High</option>
           <option value="toggle">Toggle</option>
         </select>
-        <span class="help-inline">When the event happen, you want to turn the controlled IO HIGH or LOW?</span>
+        <span class="help-inline">When the event happens, you do want to turn the controlled IO HIGH or LOW?</span>

```

Comment:
You can pull in a suggestion...my first time trying this.  If it works you should be able to just say accept suggestion...or something like that.

```suggestion
        <span class="help-inline">When the event happens, do you want to turn the controlled IO HIGH or LOW?</span>

```